.github/workflows/deploy-happy-stack.yml

```diff
@@ -162,6 +162,140 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+  run-e2e-tests:
+    timeout-minutes: 60
+    name: e2e-tests-${{ matrix.project }}-${{ matrix.shardCurrent }}-${{ matrix.shardTotal }}
+    runs-on: ubuntu-22.04
+    strategy:
+      fail-fast: false
+      matrix:
+        # Only run Chrome for now, since GHA only has 250 workers and will cancel jobs if it runs out
+        project: [chromium]
+        shardCurrent:
+          [
+            1,
+            2,
+            3,
+            4,
+            5,
+            6,
+            7,
+            8,
+            9,
+            10,
+            11,
+            12,
+            13,
+            14,
+            15,
+            16,
+            17,
+            18,
+            19,
+            20,
+          ]
+        shardTotal: [20]
```

Comment:
change the number from 30 to 20 workers, since it's equally fast and we save some workers and money as a result